intermediate_source/mario_rl_tutorial.py

```diff
@@ -348,7 +348,7 @@ def act(self, state):
 class Mario(Mario):  # subclassing for continuity
     def __init__(self, state_dim, action_dim, save_dir):
         super().__init__(state_dim, action_dim, save_dir)
-        self.memory = deque(maxlen=100000)
+        self.memory = deque(maxlen=60000) # Please change this value according to your GPU memory (current value is suitable for a GPU with 16GB RAM)
```

Comment:
@svekars 
TorchRL replay buffer won't sit on CUDA and be more efficient than a deque.
Not sure if we want to integrate it here?